### ISSUE TITLE
Why did the download of release 0.6.3 change?

### ISSUE THREAD

**Issue by Croydon** [Tessil/robin-map#37]
[Conan](https://conan.io) is a C/C++ package manager

In the Conan Center Index we have a recipe for robin-map: https://conan.io/center/tsl-robin-map/0.6.3/

The downloads of the source code are secured by sha256 checksums and are verified after every download

On June 20 version 0.6.3 was added with the sha256sum `7db3530dd6482c4a0d8f8bb52601dbdfdf3da10ac7ea9e100d4b8041a02f034e`

https://github.com/conan-io/conan-center-index/commit/ea4942a6899a544c7cd120653eb4c03a605bae12
https://github.com/conan-io/conan-center-index/pull/1950#issuecomment-647021442

Somewhen after June 20, the checksum changed to `e6654c8c2598f63eb0b1d52ff8bdf39cfcc91d81dd5d05274a6dca91241cd72f`

https://github.com/conan-io/conan-center-index/pull/2568

Why is this?

**Comment by Croydon** [comment]
@Tessil  having a look again at the release page, it looks like you re-tagged this version number to a new commit 2 days later on June 22

https://github.com/Tessil/robin-map/releases/tag/v0.6.3

I would kindly ask to not re-tag published version numbers in the future, as this causes confusion, breakages and mistrust ("is this a trustworthy change?")

**Comment by Tessil** [comment]
Hi,

Effectively, after publishing the 0.6.3 version I realized I forgot to change the version number in the `CMakeLists.txt`. I then updated the tag to the [commit](https://github.com/Tessil/robin-map/commit/622443f40544fb6a693402e69c1328d685eac939) fixing that.

I didn't take into account the package managers relying on the sha256, I should have created a 0.6.4 version instead. I'll take more care in the future when creating a new tag.
